agent/agent.go

```diff
@@ -20,7 +19,6 @@ import (
 
 	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/hcdiag/product"
-	"github.com/hashicorp/hcdiag/runner"
```

Comment:
Oh, that's a neat side effect. Didn't know that includes were the only thing left coupling the Agent to Runners.

Reply:
Yeah, I was surprised too!

Reply:
I learnt something new, nice!